python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -238,30 +242,55 @@ def read_fn():
         self.disable_node_updaters = self.config["provider"].get(
             "disable_node_updaters", False
         )
+        if self.disable_node_updaters:
+            logger.info("Node updaters are disabled.")
 
         # Disable launch config checking if true.
         # This is set in the fake_multinode situations where there isn't any
         # meaningful node "type" to enforce.
         self.disable_launch_config_check = self.config["provider"].get(
             "disable_launch_config_check", False
         )
+        if self.disable_launch_config_check:
+            logger.info("Launch config checks are disabled.")
+
+        # By default, the autoscaler launches nodes in batches asynchronously in a
+        # background thread.
+        # When the following flag is set, that behavior is disabled, so that nodes
+        # are launched in the main thread.
+        self.disable_background_launch_batch = self.config["provider"].get(
```

Comment:
Does this flag both deactivate the background thread launching and the batching? That would be good to clarify in the comment :)

Reply:
Yes, good point.